internal/flycheck/checks.go

```diff
@@ -16,6 +17,13 @@ const Port = 5500
 func Handler() http.Handler {
 	r := http.NewServeMux()
 
```

Comment:
I think the only thing that needs to change here is:

```
if os.Getenv("IS_BARMAN") != "" {
   // Add barman specific checks
} else {
  // Add pg specific checks
}
```

We can probably just hardcode the role within the role check if the `IS_BARMAN` env var is present.

Reply:
The barman role check is hardcoded!

Reply:
Right, we should just add a new condition at the top of the existing role health check:  https://github.com/fly-apps/postgres-flex/blob/master/internal/flycheck/role.go#L29 .

Reply:
Ah, got it now